README.md

```diff
@@ -5,7 +5,7 @@ Browse online:
 [![Launch JupyterLite](./book/images/jupyterbook_badge.svg 'Our JupyterBook
 website')](https://probabl-ai.github.io/calibration-cost-sensitive-learning/) [![Launch
 JupyterLite](./book/images/jupyterlite_badge.svg 'Our JupyterLite
-website')](https://probabl-ai.github.io/calibration-cost-sensitive-learning/jupyterlite)
+website')](https://probabl-ai.github.io/calibration-cost-sensitive-learning/jupyterlite/tree/notebooks)
```

Comment:
I wanted to quickly check by copy/pasting but I get a 404 page. On the principle I like (and prefer) the traditional notebook